Misc/stable_abi.toml

```diff
@@ -1628,6 +1628,8 @@
     added = '3.2'
 [function.Py_GetCompiler]
     added = '3.2'
+[function.Py_GetConfigInfo]
+    added = '3.14'
```

Comment:
The function itself is not useful without also exposing the `PY_CONFIG*` constants. And those are too CPython-specific for limited API.

Reply:
Actually the other functions used for constructing the version string were also exposed, so I assumed that mine should have been as well ... But honestly I didn't want to :/

As for the constants I think I forgot about them (first time I need to do that kind of thing...)